### PR TITLE
DE38224 - Bump d2l-activity-alignments - 20.20.4

### PR DESCRIPTION
**Pull request by lvyuan95-d2l**

### CODE REVIEW

**Comment by lvyuan95-d2l** [comment]
If a new dependency has been introduced, just add that to `package.json` and that's should be good to go, right?

**Comment by rgoessl** [comment]
> If a new dependency has been introduced, just add that to `package.json` and it should be good to go, right?

No, you have to add it to package.json, delete the package-lock, then run `npm i` to make sure that anything else using that dependency is referring to the same one

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "@polymer/esm-amd-loader": "^1.0.3",
     "@polymer/lit-element": "^0.6.1",
     "@polymer/polymer": "^3.2.0",
+    "@polymer/iron-collapse": "^3.0.1",
```
Comment (rgoessl) [inline package.json]: I think this should be removed

Comment (mpharoah-d2l) [reply package.json]: It's a new dependency, so I think this is correct?

File: package-lock.json [hunk 2]
```diff
@@ -4176,7 +4176,7 @@
       }
     },
     "d2l-activity-alignments": {
-      "version": "github:Brightspace/d2l-activity-alignments#90107c3d532f67003a4afb0748430dedd8193d71",
+      "version": "github:Brightspace/d2l-activity-alignments#ddeae1fc858e9f0ccaf0e6eae2559e2f22cfecd9",
```
Comment (mpharoah-d2l) [inline package-lock.json]: This is using the same version as the 20.20.5 one. I this correct, or should we be targeting a different branch?

File: package.json (hunk 1 above)
Comment (rgoessl) [reply package.json]: Other repos use it, so it's not new to BSI

File: package-lock.json (hunk 2 above)
Comment (lvyuan95-d2l) [reply package-lock.json]: I think we want those fixes on both 20.20.4 and master, not sure if anything in cert 20.20.4 will be merged in master automatically?

**Comment by lvyuan95-d2l** [comment]
So the weird thing is that `@polymer/iron-collapse` has been used in other components as well as in `package-lock.json`, but not in `package.json`

Should I just bump the `d2l-activity-alignments` and add `@polymer/iron-collapse` to its `requires` in `package-lock.json`?

**Comment by rgoessl** [comment]
> Should I just bump the `d2l-activity-alignments` and add `@polymer/iron-collapse` to its `requires` in `package-lock.json`?

If you bump it using `npm i` it should automatically update all dependencies. But you would have to delete the package-lock first

**Pull request merged**